variation-commons-mongodb/src/test/java/uk/ac/ebi/eva/commons/mongodb/services/VariantWithSamplesAndAnnotationServiceTest.java

```diff
@@ -102,5 +102,11 @@ public void testFindChromosomeBoundaries() {
         assertNull(service.findChromosomeHighestReportedCoordinate("11", Arrays.asList("PRJEB5870")));
     }
 
+    @Test
+    public void testCountTotalNumberOfVariants() {
```

Comment:
This test should check the specific number that applies to the test dataset, otherwise a wrong positive number could be returned and we wouldn't notice.

Reply:
@cyenyxe You're right! Changed :)